linchpin/provision/roles/inventory_gen/tasks/main.yml

```diff
@@ -25,9 +25,14 @@
   set_fact:
     inventory_layout: "{{ layout_file | default( default_layouts_path+'/'+'openshift-3node-cluster.yml') | ordered_yaml }}"
 
+- name: "Generate topology_name"
+  set_fact:
+    topology_name: "{{ topology_file | strip_extension }}"
```

Comment:
I don't think this is what we want to do. Rather, we want to base it upon the topology_name variable, just as it is for resources.